utils/src/main/java/software/amazon/awssdk/utils/IoUtils.java

```diff
@@ -22,10 +22,12 @@
 import java.nio.charset.StandardCharsets;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.annotations.SdkProtectedApi;
 
 /**
  * Utilities for IO operations.
  */
+@SdkProtectedApi
```

Comment:
Public

Reply:
Before GA, sure. It's not ready to be public right now, though.